SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/Spatial2WorkerMap.cpp

```diff
@@ -26,6 +28,11 @@ void USpatial2WorkerMap::CreateCustomContentForMap()
 	// that they can demonstrate they work in all situations
 	AddActorToLevel<ACrossServerAndClientOrchestrationTest>(CurrentLevel, Server2Pos);
 	AddActorToLevel<ACrossServerAndClientOrchestrationTest>(CurrentLevel, Server1Pos);
+	AddActorToLevel<AEntityInteractionTestActor>(CurrentLevel, Server2Pos)->Index = 0;
```

Comment:
In theory AddActorToLevel can fail?

Reply:
Maybe, but what would be the point of a graceful failure ?

Reply:
I'd prefer a graceful failure to a null pointer write?

Reply:
But a graceful failure would not force someone to fix it.

Reply:
By graceful I meant throw a test error rather than overwrite random memory - at least the test would fail.

Or add a new function called 'SafeAddToActorToLevel' or similar name - which is designed to never fail.

Reply:
This code does not run in a test but in the command that generate the maps for the test.
Actually, I do not see any reason why we would allow AddActorToLevel (which is ours) to fail. Could make it check() itself.

Reply:
Actually, it does a CastChecked, so I'm going to make it return a Ref.

Reply:
Done in https://github.com/spatialos/UnrealGDK/pull/3012